sympy/core/tests/test_compatibility.py

```diff
@@ -4,6 +4,7 @@
 from sympy.testing.pytest import raises
 
 from sympy.abc import x
+import numpy as np
```

Comment:
You need to allow this test to fail if numpy is not installed. Take a look at how it is done in sympy/utilities/tests/test_lambdify.py

Reply:
Alright, I'll look into it. Thanks.